### PR TITLE
zh-cn/web/http clear up a tag

### PR DESCRIPTION
**Pull request by fwqaaq**

### CODE REVIEW

**Comment by Jack-Zhang-1314** [comment]
>\<a class="haloword-button" id="haloword-pron" title="发音">\</a>

* delete only the title, or delete the entire a tag

File: files/zh-cn/web/http/session/index.html [hunk 1]
```diff
@@ -104,7 +104,7 @@ <h3 id="响应示例">响应示例</h3>
 Server: Apache/2.2.3 (Red Hat)
 Content-Type: text/html; charset=iso-8859-1
 Date: Sat, 09 Oct 2010 14:30:24 GMT
-Location: <a class="linkification-ext" href="../../../../" title="Linkification: https://developer.mozilla.org/">https://developer.mozilla.org/</a> <strong><em>(目标资源的新地址，服务器期望用户代理去访问它</em><em>)</em></strong>
+Location: <a href="../../../../">https://developer.mozilla.org/</a> <strong><em>(目标资源的新地址，服务器期望用户代理去访问它</em><em>)</em></strong>
```
Comment (yin1999) [inline files/zh-cn/web/http/session/index.html:107]:
```suggestion
Location: https://developer.mozilla.org/ （目标资源的新地址，服务器期望用户代理去访问它）
```

File: files/zh-cn/web/http/session/index.html [hunk 2]
```diff
@@ -118,7 +118,7 @@ <h3 id="响应示例">响应示例</h3>
 &lt;title&gt;301 Moved Permanently&lt;/title&gt;
 &lt;/head&gt;&lt;body&gt;
 &lt;h1&gt;Moved Permanently&lt;/h1&gt;
-&lt;p&gt;The document has moved &lt;a href="<a class="linkification-ext" href="../../../../" title="Linkification: https://developer.mozilla.org/">https://developer.mozilla.org/</a>"&gt;here&lt;/a&gt;.&lt;/p&gt;
+&lt;p&gt;The document has moved &lt;a href="<a href="../../../../">https://developer.mozilla.org/</a>"&gt;here&lt;/a&gt;.&lt;/p&gt;
```
Comment (yin1999) [inline files/zh-cn/web/http/session/index.html:121]:
```suggestion
&lt;p&gt;The document has moved &lt;a href="https://developer.mozilla.org/"&gt;here&lt;/a&gt;.&lt;/p&gt;
```

**Review by yin1999 — APPROVED** [review]
here are two suggestions about code example

File: files/zh-cn/web/http/session/index.html (hunk 2 above)
Comment (Jack-Zhang-1314) [reply files/zh-cn/web/http/session/index.html]:
```
&lt;p&gt;The document has moved &lt;a href="<a href="../../../../">https://developer.mozilla.org/</a>"&gt;here&lt;/a&gt;.&lt;/p&gt;
```

* It's too hidden

**Pull request merged**